src/one/nio/mem/MappedFile.java

```diff
@@ -170,6 +170,12 @@ public static long map(RandomAccessFile f, int mode, long start, long size) thro
             if (map0 != null) {
                 return (long) map0.invoke(f.getChannel(), mode, start, size, false);
             }
+            // Since 19 JDK has an extra 'file descriptor' argument
+            map0 = JavaInternals.getMethod(
+                cls, "map0", FileDescriptor.class, int.class, long.class, long.class, boolean.class);
+            if (map0 != null) {
+                return (long) map0.invoke(f.getChannel(), f.getFD(), mode, start, size, false);
```

Comment:
Shouldn't file descriptor go first as you mentioned in the description?
```suggestion
                return (long) map0.invoke(f.getFD(), f.getChannel(), mode, start, size, false);
```

Reply:
yes, fixed